ansible/roles/baselayout/vars/main.yml

```diff
@@ -70,6 +70,10 @@ packages: {
     'gcc-c++,sudo',
   ],
 
+  rhel72_s390x: [
+    'at-spi2-core-devel',
+  ],
```

Comment:
Are these additive (i.e. on `rhel72_s390x` both `rhel72` and `rhel72_s390x` apply) or does one take preference over the other?

Asking because the comment on L34 says:
> ```
> # you can either add os family or os to this list (see smartos)
> # but the playbook chooses os over family - not both
> ```

Reply:
I just copied the example on https://github.com/nodejs/build/blob/master/ansible/roles/baselayout/vars/main.yml#L43